src/test/java/com/example/demo/FileReaderTest.java

```diff
@@ -0,0 +1,51 @@
+package com.example.demo;
+
+import com.example.demo.exceptions.ReadFileException;
+import com.example.demo.service.FileReaderService;
+import com.example.demo.service.impl.FileReaderServiceImpl;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderTest {
+    private static final String PATH_TO_VALID_TEST_FILE = "src/main/resources/test_file.csv";
+    private static final String WRONG_PATH = "src/main/resources/file.csv";
```

Comment:
```suggestion
    private static final String PATH_TO_VALID_TEST_FILE = "src/test/resources/test_file.csv";
    private static final String WRONG_PATH = "src/test/resources/file.csv";
```

resources for testing should better lay here

Reply:
done